src/Code.js

```diff
@@ -469,7 +469,12 @@ function MBCUSTOMQUERY(events, groupBy, orderBy, limit, offset) {
 
   // turn the array of objects into a flat array
   const objArr = results.result.rows;
-  return objectArrayToArray(objArr);
+  const headersPreset = Array(payload.events[0].select.length);
```

Comment:
Could `events[0]` ever be `null`?

Reply:
Yes, it could be null but `throw new Error()` will happen before

https://github.com/curvegrid/multibaas-for-google-sheets/blob/15bfb634aaeae8591eb0e0d136bf3c7ba99fbf23/src/library/Build.js#L136-L151